pkg/worker/work_runner.go

```diff
@@ -38,7 +39,8 @@ func Run(work *model.WorkRequest) error {
 	// Exec script with args...
 	cmd := exec.Command(binary, work.Payload)
 	// with env variables...
-	cmd.Env = append(os.Environ(), "HOOK_ID=" + work.ID , work.Args...)
+	workEnv := append(os.Environ(), fmt.Sprintf("HOOK_ID=%d", work.ID))
```

Comment:
Can you use lower-case in order to follow bash conventions (upper-case should be for parent environnement variables)
```suggestion
	workEnv := append(os.Environ(), fmt.Sprintf("hook_id=%d", work.ID))
```